### PR TITLE
content for definitions

### PR DESCRIPTION
**Pull request by dalvarez83**
**Created document for definitions for data science/AI toolkit**

** Next steps:**
- need to configure Figure 1 into document

### CODE REVIEW

File: content/01-definitions/definitions.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+**Natural Language Processing** (NLP) heavily leverages Machine Learning as well as Deep Learning to allow computers to interact with humans in their own natural language whether written or oral. NLP applications attempt to understand natural human communication, either written or spoken, and communicate in return with us using similar, natural language.  
+
+**Robotic Process Automation** (RPA) is a type of automation that mimics the activity of a human when they carry out a structured, repetitive task or process, little judgement. It is generally rule based, can’t learn outside parameters, so not considered AI; however, RPA can be built with learning modules on complex tasks.  
+
+Figure 1 diagram below depicts the AI ecosystem and the relationship between AI, ML, Deep Learning, NLP and RPA. 
+
+![Figure 1: The AI ecosystem](../../static/images/ai-ecosystem.png)
```
Comment (justwheel) [inline content/01-definitions/definitions.md]: There are two ways to link images. I prefer the first method:

1. Upload the image into the git repository in a path that matches the name of the file, e.g. `static/01-definitions/definitions/ai-ecosystem.png`. Link using the file name (the suggestion offered in this comment).
2. Upload the file anywhere in the `static/` directory. All links must be hard-coded with paths, e.g. `/ooi-toolkit-ds/images/ai-ecosystem.png`.

```suggestion
![Figure 1: The AI ecosystem](ai-ecosystem.png)
```

File: content/01-definitions/definitions.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Definitions"
+icon: "ti-search"
```
Comment (justwheel) [inline content/01-definitions/definitions.md]: This line you only need for an `_index.en.md` file. In the index file, you use that to initialize and create the category. Without an index page, your category will not appear on the site homepage. This page here will publish as `https://unicef.github.io/ooi-toolkit-ds/01-definitions/definitions/`

**Review by justwheel — COMMENTED** [review]
This looks good to me for content. I shared a tip below on how to add images/figures into a document. You'll also need to upload the image to the pull request.

Also, this category won't appear on the site homepage without an index file, which might be needed before merging this. See in-line comment for more details.

File: content/01-definitions/definitions.md (hunk 1 above)
Comment (dalvarez83) [reply content/01-definitions/definitions.md]: Thanks. I uploaded image to the path `static/images/01-definitions/definitions/ai-ecosystem.png` through a commit

Comment (dalvarez83) [reply content/01-definitions/definitions.md]: Should I create a separate pull request?

File: content/01-definitions/definitions.md (hunk 2 above)
Comment (dalvarez83) [reply content/01-definitions/definitions.md]: Do you suggest I remove this descriptive annotation from the `definitions.md` file? Should I create a separate index file?

Comment (justwheel) [reply content/01-definitions/definitions.md]: The description annotation is useful. This is used for OpenGraph data, or what pops up in the box when you share a link on social media or a messaging app. It should be a sentence or two summary of the article contents.

File: content/01-definitions/definitions.md (hunk 1 above)
Comment (justwheel) [reply content/01-definitions/definitions.md]: Looks like I gave you bad advice about where to store the files. I'll push a new commit to the Pull Request with the fix and explain it in a new comment.

**Comment by justwheel** [comment]
See my commit e0ebf6081062f7f79caafa3119ff36a422580d7d. I corrected my mistake about where the image needs to go; it should go in the same directory as the category where it will be used. That commit also gets the C.I. pipeline to pass, which was failing because the image link was incorrect.

@dalvarez83 Is this ready to merge?

**Comment by dalvarez83** [comment]
Appears ready to merge.

**Comment by justwheel** [comment]
There was a bug with how the CI pipeline deployed the changes. I fixed it in commit d91c3ebb3528527460d9e7686857130af58452d7. The page is now published in the toolkit:

https://unicef.github.io/ooi-toolkit-ds/01-definitions/

**Pull request merged**